execute/execute.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import json
+import jsonschema
+import time
+
+from pathlib import Path
+
+from console_access_api.aitrios_console import AitriosConsole
+
+parser = argparse.ArgumentParser(
+    description="Launches execution on a camera configured in AITRIOS Console",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+
+parser.add_argument(
+    "--aitrios_secrets",
+    "-s",
+    type=str,
+    dest="aitrios_secrets",
+    help=f"The path to your AITIRIOS console configuration file",
+    required=True,
+)
+parser.add_argument(
+    "--device_name",
+    "-D",
+    type=str,
+    dest="device_name",
+    help=f"Device Name to search for",
+    required=True,
+)
+
+
+def load_configuration_file(configuration_path: Path, schema_path: Path):
+    """
+    Parse a secrects configuration file, validates against schema
+    """
+    # Load configuration file
+    with open(configuration_path, "r") as f:
+        configuration = json.load(f)
+
+    # Load configuration schema file
+    with open(schema_path, "r") as f:
+        json_schema = json.load(f)
+
+    # Validate configuration
+    jsonschema.validate(configuration, json_schema)
+    return configuration
+
+
+def create_client(aitrios_secrets):
+    """
+    Create a connection object
+    """
+    json_load = load_configuration_file(
+        aitrios_secrets, "./json_schemas/console_configuration_schema.json"
+    )
+    return AitriosConsole(
+        json_load["console_endpoint"],
+        json_load["client_id"],
+        json_load["client_secret"],
+        json_load["portal_authorization_endpoint"],
+    )
+
+
+def jsonify(data):
+    """
+    Dumnp data as a JSON element, pretty formated
+    """
+    return json.dumps(data, indent=4)
+
+if __name__ == "__main__":
+    print("Initiating Console connection")
+    args = parser.parse_args()
+    client_obj = create_client(args.aitrios_secrets)
+
+    # 1. Get a device information
+    print(f"Looking for device {args.device_name}")
+    request = client_obj.GetDevices(
+        connectionState="connected", device_name=args.device_name
+    )
+    devices = request["devices"]
+    if not devices or len(devices) == 0:
+        connected_devices = client_obj.GetDevices(connectionState="connected")
+        print(f"Connected Devices")
```

Comment:
Unnecessary f-string

```suggestion
        print("Connected Devices")
```